cmd/describe.go

```diff
@@ -161,3 +161,10 @@ func (d description) XML(w io.Writer) error {
 func (d description) YAML(w io.Writer) error {
 	return yaml.NewEncoder(w).Encode(d)
 }
+
+func (d description) URL(w io.Writer) error {
+	if len(d.Routes) > 0 {
+		fmt.Fprint(w, d.Routes[0])
```

Comment:
```suggestion
		fmt.Fprintf(w, "%s\n", d.Routes[0])
```

Reply:
I was thinking the linefeed might mess up `curl $(func describe -o url)` but it doesn't seem to. Thanks!